test/test_quantized.py

```diff
@@ -1222,18 +1222,18 @@ class TestQNNPackOps(TestCase):
                                           zero_point_min=0,
                                           zero_point_max=0)))
     def test_qnnpack_relu(self, X):
-        X, (scale, zero_point, torch_type) = X
-        relu = torch.ops.quantized.qnnpack_relu
-
-        X = torch.from_numpy(X)
-        Y = X.clone()
+        with enable_mobile_quantized_engine():
```

Comment:
Similar comments to conv/linear qnnpack tests:  We should ensure that all backends share the same tests and have consistent numerics. Can you merge this test with the existing relu test?

Reply:
Discussed offline, will merge in a separate PR.